### PR TITLE
Update ir.m3u

### PR DESCRIPTION
**Pull request by data14a**
Replaced one dead link for AFN TV.
Renamed six channels from Persiana TV group and added four new channels.
Updated TV logos from GEM TV group and added eight new channels.

### CODE REVIEW

File: channels/ir.m3u [hunk 1]
```diff
@@ -4,7 +4,7 @@ http://51.158.190.97/hls/stream.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="4U TV" tvg-language="Persian" tvg-logo="https://i.imgur.com/Icob3E3.jpg" group-title="Entertainment",4U TV
 http://live.4ulive.ir/
 #EXTINF:-1 tvg-id="" tvg-name="Afghan TV" tvg-language="Persian" tvg-logo="https://www.parsatv.com/index_files/channels/afghantv.jpg" group-title="",AFN TV
-https://unirtmp.tulix.tv/tintv2/tintv2/playlist.m3u8
```
Comment (freearhey) [inline channels/ir.m3u:7]: Link works fine. No need to replace it.

File: channels/ir.m3u [hunk 2]
```diff
@@ -33,25 +33,39 @@ https://bozztv.com/36bay2/gin/giniko_tvbox-cine-series_800kb_36bay2/tracks-v1a1/
...
 http://d2e40kvaojifd6.cloudfront.net/stream/gem_drama/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM Junior" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-junior.png?1212" group-title="Entertainment",GEM Junior
+#EXTINF:-1 tvg-id="" tvg-name="GEM Film" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_film.png?13" group-title="Entertainment",Gem Film
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_film/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Fit" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_fit.png?13" group-title="Entertainment",Gem Fit
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_fit/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Food" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_food.png?13" group-title="Entertainment",Gem Food
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_food/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Junior" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemjunior.png?13" group-title="Entertainment",GEM Junior
```
Comment (freearhey) [inline channels/ir.m3u:52]: Links do not work.

File: channels/ir.m3u [hunk 3]
```diff
@@ -33,25 +33,39 @@ https://bozztv.com/36bay2/gin/giniko_tvbox-cine-series_800kb_36bay2/tracks-v1a1/
...
-http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-name="GEM TV" tvg-language="Persian" tvg-logo="http://www.gemonline.tv/Assets/channels-box/gem-tv.png?1212" group-title="Entertainment",GEM TV
+#EXTINF:-1 tvg-id="" tvg-name="GEM Life" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_life.png?13" group-title="Entertainment",Gem Life
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_life/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Nature" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_nature.png?13" group-title="Entertainment",Gem Nature
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_nature/playlist_960x540_2000k.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="GEM Property" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_property.png?13" group-title="Entertainment",Gem Property
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_property/playlist_960x540_2000k.m3u8
```
Comment (freearhey) [inline channels/ir.m3u:61]: Links do not work.

File: channels/ir.m3u [hunk 4]
```diff
@@ -33,25 +33,39 @@ https://bozztv.com/36bay2/gin/giniko_tvbox-cine-series_800kb_36bay2/tracks-v1a1/
...
+#EXTINF:-1 tvg-id="" tvg-name="GEM Series" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gemseries.png?13" group-title="Entertainment",Gem Series
+http://d2e40kvaojifd6.cloudfront.net/stream/gem_series/playlist.m3u
+#EXTINF:-1 tvg-id="" tvg-name="GEM Travel" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/gem_travel.png?13" group-title="Entertainment",Gem Travel
+https://d2e40kvaojifd6.cloudfront.net/stream/gem_travel/playlist_960x540_2000k.m3u8
```
Comment (freearhey) [inline channels/ir.m3u:65]: Links do not work.

File: channels/ir.m3u [hunk 5]
```diff
@@ -127,6 +141,8 @@ http://d2e40kvaojifd6.cloudfront.net/stream/maxx/playlist.m3u8
 https://d2e40kvaojifd6.cloudfront.net/stream/mifa/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="Mihan TV" tvg-language="Persian" tvg-logo="http://mihantv.com/wp-content/themes/mihantv/images/logo.png" group-title="News",Mihan TV
 https://iptv.mihantv.com/live/playlist1/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="Modern Economy" tvg-language="Persian" tvg-logo="https://www.gemonline.tv/Assets/channels-box/modern_economy.png?13" group-title="Entertainment",Modern Economy
+https://d2e40kvaojifd6.cloudfront.net/stream/modern_enonomy/playlist_960x540_2000k.m3u8
```
Comment (freearhey) [inline channels/ir.m3u:145]: Link do not work.

**Review by freearhey — CHANGES_REQUESTED** [review]
Please check the marked links again.

File: channels/ir.m3u (hunk 5 above)
Comment (data14a) [reply channels/ir.m3u:145]: I just checked with https://castr.io/hlsplayer and it is working

File: channels/ir.m3u (hunk 4 above)
Comment (data14a) [reply channels/ir.m3u:65]: I just checked with https://castr.io/hlsplayer and it is working

File: channels/ir.m3u (hunk 3 above)
Comment (data14a) [reply channels/ir.m3u:61]: I just checked with https://castr.io/hlsplayer and they are working

File: channels/ir.m3u (hunk 2 above)
Comment (data14a) [reply channels/ir.m3u:52]: I just checked with https://castr.io/hlsplayer and they are working

File: channels/ir.m3u (hunk 1 above)
Comment (data14a) [reply channels/ir.m3u:7]: when I checked yesterday it was not working

File: channels/ir.m3u (hunk 5 above)
Comment (freearhey) [reply channels/ir.m3u:145]: This is what I see on https://castr.io/hlsplayer
Screenshot 2020-09-12 at 14 46 31

Comment (data14a) [reply channels/ir.m3u:145]: I am not sure what is going on but try this stream which is lower in quality.
https://d2e40kvaojifd6.cloudfront.net/stream/modern_economy/playlist_640x360_800k.m3u8

Comment (freearhey) [reply channels/ir.m3u:145]: Same thing.

Comment (data14a) [reply channels/ir.m3u:145]: I just checked with VLC and https://castr.io/hlsplayer and both are playing.

Comment (freearhey) [reply channels/ir.m3u:145]: This is what I see in VLC:
Screenshot 2020-09-12 at 20 38 41

Comment (data14a) [reply channels/ir.m3u:145]: vlcsnap-2020-09-12-13h50m42s894
This is a snapshot of my VLC player. I also loaded the m3u in firestick and plays in there with tivi mate

Comment (freearhey) [reply channels/ir.m3u:145]: Then I assume the broadcast is geoblocked.

Comment (data14a) [reply channels/ir.m3u:145]: I don't think so because all the Gem channels are from the same server.

Comment (bertperrisor) [reply channels/ir.m3u:145]: Working on Singapore IP

Screenshot 2020-09-14 at 2 10 06 PM

Comment (freearhey) [reply channels/ir.m3u:145]: @data14a But the fact is that all channels from this server give out an error.

Comment (data14a) [reply channels/ir.m3u:145]: I just checked the streams from this server using VLC in windows 7 and also in windows 10. Checked with tivimate on firestick and with https://www.jwplayer.com/developers/stream-tester/. They all play fine so the error might be on your side.

Comment (freearhey) [reply channels/ir.m3u:145]: Maybe you are right and these links do not work only for me.

Comment (data14a) [reply channels/ir.m3u:145]: I am not sure what to do at this point. If you are not going to approve this change you can close the request.

**Pull request closed without merging**